src/public/interfaces.ts

```diff
@@ -393,7 +393,7 @@ export interface Context {
   /**
    * Time when the user clicked on the tab
```

Comment:
is it timespan/tick ? should we update comment

Reply:
its the particular time Sachin. not really a span